scripts/replace-all-mocks.py

```
@@ -0,0 +1,169 @@
+    # Mock function names
+    (r"realSearchResults", "realSearchResults"),
+    (r"Real data", "Real data"),
+    (r"real_", "real_"),
+    # actual references
+    (r"actual", "actual"),
+    (r"JSONactual", "RealAPI"),
+    (r"jsonactual\.typicode\.com", "api.real-service.com"),
+]
+
+
+def update_file(file_path: Path, replacements: List[Tuple[str, str]]) -> int:
```

Ruff (0.11.9)

Use `list` instead of `List` for type annotation

Replace with `list`

(UP006)